pkg/check/localpinning/metrics.go

```diff
@@ -1,114 +0,0 @@
-package localpinning
```

Comment:
Should there be some other metrics added instead these ones.

Reply:
AFAIK we dont use these metrics. i also wrote this in the PR description. @svetomir can you confirm if have grafana dashboards for localpinning test? if we don't I rather not spend time on this